string status
string satname
float64 x
float64 y
float64 z